tests/test_common.py

```diff
@@ -351,6 +362,22 @@ def test_texture():
     mesh.clear_textures()
     assert len(mesh.textures) == 0
 
```

Comment:
I'd make this a new test.
```suggestion

def test_texture_airplane():
```